### ISSUE TITLE
npm ERR! Test failed

### ISSUE THREAD

**Issue by slash3b** [tldr-pages/tldr#754]
Hi.
Could somebody let me know what am I doing wrong? Just pulled repo to local machine, followed Contribution guidelines. On `npm test ` I got this error:    

```
> tldr@1.0.0 test /var/www/html/tldr
> markdownlint pages/**/*.md && tldr-lint ./pages                                                                                                                    
/var/www/html/tldr/node_modules/tldr-lint/lib/tldr-lint.js:115                             
  str += util.format(`# ${parsedPage.title}`);                                             
                     ^                                                                     
SyntaxError: Unexpected token ILLEGAL                                                      
    at exports.runInThisContext (vm.js:73:16)                                              
    at Module._compile (module.js:443:25)                                                  
    at Object.Module._extensions..js (module.js:478:10)                                    
    at Module.load (module.js:355:32)                                                      
    at Function.Module._load (module.js:310:12)                                            
    at Module.require (module.js:365:17)                                                   
    at require (module.js:384:17)                                                          
    at Object.<anonymous> (/var/www/html/tldr/node_modules/tldr-lint/lib/tldr-lint-cli.js:3
:14)                                                                                       
    at Module._compile (module.js:460:26)                                                  
    at Object.Module._extensions..js (module.js:478:10)                                    
npm ERR! Test failed.  See above for more details.     

```

**Comment by igorshubovych** [comment]
@rubenvereecken it seems like this is a problem in ltdr-lint

**Comment by rubenvereecken** [comment]
It it known. Apparently I used some arcane Node feature that is at least Node 4.x and definitely present in 5.x. I have not cared so far to switch syntax.

**Comment by igorshubovych** [comment]
/me Submitted a PR to fix it.

**Comment by slash3b** [comment]
Thank you guys, my Node was really outdated(v0.12), I got latest v5.5.0 and the error is gone.

**Comment by 85pando** [comment]
I also got a lint error:

```
$ npm test

> tldr@1.0.0 test /home/pandora/Documents/Seafile/Repos/tldr
> markdownlint pages/**/*.md && tldr-lint ./pages


/home/pandora/Documents/Seafile/Repos/tldr/node_modules/tldr-lint/lib/tldr-lint.js:115
  str += util.format(`# ${parsedPage.title}`);
                     ^
SyntaxError: Unexpected token ILLEGAL
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/home/pandora/Documents/Seafile/Repos/tldr/node_modules/tldr-lint/lib/tldr-lint-cli.js:3:14)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
npm ERR! Test failed.  See above for more details.
```

The problem seemed to be the usage of ``` ` ``` instead of ``` ' ``` (after replacing them it works). Looking at the error message from above, where there seems to be no text delimiter at all, I wonder if the exact script depends on the client somehow?

**Comment by slash3b** [comment]
sorry @85pando, it is just my badly formatted message, I've updated it. It is the same error.

**Comment by 85pando** [comment]
All right then! ☻

**Comment by ostera** [comment]
@rubenvereecken your syntax is good and future-friendly! It's a [Template String](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/template_strings)

I personally much rather we used Babel and wrote ES6, and specify the npm files in package.json to be the built artifacts (which naturally shouldn't be part of the repo).

**Comment by igorshubovych** [comment]
FYI, this is already addressed
https://github.com/tldr-pages/tldr-lint/pull/2

**Comment by rubenvereecken** [comment]
I've been wanting to write ES6 for a time now, looks like I mistakenly used a feature. Cheers @igorshubovych!

**Comment by igorshubovych** [comment]
Closing this. Thanks, everybody!
